app/views/layouts/_sidebar.html.slim

```diff
@@ -1,4 +1,3 @@
 .govuk-grid-column-one-third
-  .govuk-header__container.govuk-width-container
-  div class="govuk-!-padding-bottom-4"
-  = yield
\ No newline at end of file
+  aside.app-related-items role="complementary"
```

Comment:
The alignment of the sidebar on the start page is skewiff:

<img width="366" alt="Screenshot 2022-10-12 at 15 51 37" src="https://user-images.githubusercontent.com/113888736/195376463-2c8199c6-944a-4b75-849b-99fdd85c4bf5.png">

The blue box header extends out further to the right than the thick blue page header